Header header

float32 throttle_cmd
float32 throttle_output
bool enabled

#do not have worry about those
#Status
bool override #Driver override
bool driver #Driver activity
bool timeout # Command timeout


